### ISSUE TITLE
testing: investigate apparent cmd/go CI flake

### ISSUE THREAD

**Issue by cueckoo** [cue-lang/cue#727]
_Originally opened by @myitcv in https://github.com/cuelang/cue/issues/727_

```
go proxy: no archive example.com v1.0.0
--- FAIL: TestScript (0.21s)
    --- FAIL: TestScript/get_go_types (4.21s)
        testscript.go:382: 
            # Test that a get go works for various "interesting" types,
            # including those special cases that result in _ and string
            # that are specifically mentioned in help get go.
            #
            # This test does not exercise with/without --local. Instead
            # that is left for other tests, so that this test can focus
            # on the various rules of get go.
            # cue get go (3.740s)
            > cue get go --local ./...
            [stderr]
            could not load Go packages:
            	./...: [-: pattern ./...: stat $WORK\tmp\go-build655607887: directory not found]
            [exit status 1]
            FAIL: testdata\script\get_go_types.txt:10: unexpected command failure
            
FAIL
```

Instances:

### Windows, `go1.15`

* https://github.com/cuelang/cue/runs/1837593473?check_suite_focus=true 
* https://github.com/cuelang/cue/runs/1844129547?check_suite_focus=true 
* https://github.com/cuelang/cue/runs/1845055663?check_suite_focus=true

**Comment by cueckoo** [comment]
_Original reply by @myitcv in https://github.com/cuelang/cue/issues/727#issuecomment-774542278_

Raised https://github.com/golang/go/issues/44137 with what we know so far

**Comment by cueckoo** [comment]
_Original reply by @myitcv in https://github.com/cuelang/cue/issues/727#issuecomment-831164902_

Resolved per golang/go#44137
